### DISCUSSION TITLE
可以添加一个剩余流量消息推送功能吗？

### DISCUSSION THREAD

**Discussion by cfj521** [vernesong/OpenClash#3734] [Ideas]
如题。经常需要进管理页面（甚至有时候管理页面也无法正确获得剩余流量信息），或者登录梯子网站去查看，很麻烦，一旦用超了，就更麻烦。。
所以，如题，开发组人员能不能增加一个剩余流量信息的推送功能，当然比如clash运营异常，所选节点异常等也可以加进去。推送的话可以用强制网页重定向、或者微信、钉钉之类的消息推送。
谢谢开发组

**Comment by ghost** [comment]
你好，没有组，就v大一个人

请问`甚至有时候管理页面也无法正确获得剩余流量信息`的原因是什么呢？
